lutris/gui/application.py

```diff
@@ -627,6 +682,104 @@ def print_steam_folders(self, command_line):
             for path in steamapps_paths[platform]:
                 self._print(command_line, path)
 
+    def print_runners(self):
+        runnersName = get_runner_names()
+        print("Runners:")
+        for name in runnersName:
+            print(name)
+
+    def print_wine_runners(self):
+        runnersName = get_runners("wine")
+        for i in runnersName["versions"]:
+            if i["version"]:
+                print(i)
+
+    def install_runner(self, runner):
+        if runner.startswith("lutris"):
+            self.install_wine_cli(runner)
+        else:
+            self.install_cli(runner)
+
+    def uninstall_runner(self, runner):
+        if "wine" in runner:
+            print("Are sure you want to delete Wine and all of the installed runners?[Y/N]")
+            ans = input()
+            if ans.lower() in ("y", "yes"):
+                self.uninstall_runner_cli(runner)
+            else:
+                print("Not Removing Wine")
+        elif runner.startswith("lutris"):
+            self.wine_runner_uninstall(runner)
+        else:
+            self.uninstall_runner_cli(runner)
+
+    def install_wine_cli(self, version):
+        """
+        Downloads wine runner using lutris -r <runner>
+        """
+
+        WINE_DIR = os.path.join(settings.RUNNER_DIR, "wine")
+        runner_path = os.path.join(WINE_DIR, f"{version}{'' if '-x86_64' in version else '-x86_64'}")
+        if os.path.isdir(runner_path):
+            print(f"Wine version '{version}' is already installed.")
+        else:
+            from lutris.gui.dialogs import ErrorDialog
+            from lutris.gui.dialogs.download import simple_downloader
+            try:
+                runner = import_runner("wine")
+                runner().install(downloader=simple_downloader, version=version)
+                print(f"Wine version '{version}' has been installed.")
+            except RunnerInstallationError as ex:
```

Comment:
```suggestion
            except (InvalidRunner, RunnerInstallationError) as ex:
```
To cover the possible exception raised by import_runner().